src/modules/fancyzones/dll/dllmain.cpp

```diff
@@ -129,7 +138,7 @@ class FancyZonesModule : public PowertoyModuleIface
     // Returns if the powertoy is enabled
     virtual bool is_enabled() override
     {
-        return (m_app != nullptr);
+        return !disableFancyZonesModuleCore && m_app != nullptr;

```

Comment:
Is this needed? If we are calling disable() when setting disableFancyZonesModuleCore, it should set m_app to null?

Reply:
This variable is used to make early exit in every call to `enable()`, which essentially comes after interface is constructed.